docs/source/data_analysis/HPC-module-SLEAP.md

```diff
@@ -76,19 +98,19 @@ After that, you can follow the [rest of the SLEAP installation guide](https://sl
 
 :::{tab-item} Windows and Linux
 ```{code-block} console
-$ conda create -y -n sleap -c conda-forge -c nvidia -c sleap -c anaconda sleap=1.3.1
+$ conda create -y -n sleap -c conda-forge -c nvidia -c sleap -c anaconda sleap=1.3.3
```

Comment:
It feels a bit of an unnecessary burden to ensure that this section is in sync with SLEAP's docs.

I think we can remove most of it and keep until: 
> We recommend following the official [SLEAP installation guide](https://sleap.ai/installation.html).

I would keep your clarification at the end, I like it, but maybe rephrase a bit to include why:
> To minimise the risk of issues due to incompatibilities between versions, ensure the version of your local installation of SLEAP matches the one you plan to load in the cluster.

Reply:
agreed, you're totally right.